qbittorrent/Client.py

```diff
@@ -801,7 +801,7 @@ def toggle_alternative_speed(self):
         """
         Toggle alternative speed limits.
         """
-        return self._get("transfer/toggleSpeedLimitsMode")
+        return self._post("transfer/toggleSpeedLimitsMode")
```

Comment:
This will fail if you don't fill the `data` parameter

```suggestion
        return self._post("transfer/toggleSpeedLimitsMode", data={})
```